src/converge.js

```diff
@@ -33,7 +33,8 @@ var pluck = require('./pluck');
  *      R.converge(add3, [multiply, add, subtract])(1, 2); //=> 4
  */
 module.exports = _curry2(function converge(after, fns) {
-  return curryN(Math.max.apply(Math, pluck('length', fns)), function() {
+  var arity = fns.length ? Math.max.apply(Math, pluck('length', fns)) : 0;
```

Comment:
Math.max returns -Infinity for zero arguments, which is why converge didn't work for an empty functions list,